src/rebuilder/mod.rs

```diff
@@ -17,5 +18,6 @@ where
     type Error: std::error::Error;
 
     async fn by_aggregate_id(&self, executor: Self::Executor) -> Result<(), Self::Error>;
+    async fn just_one_aggregate(&self, aggregate_id: Uuid, executor: Self::Executor) -> Result<(), Self::Error>;
     async fn all_at_once(&self, executor: Self::Executor) -> Result<(), Self::Error>;
 }
```

Comment:
Can we add docstrings to explain what these methods actually do?
I'm not familiar with the rebuilder and I cannot understand what's the difference between by_id and all_at_once